spec/parsers/bulkrax/csv_parser_spec.rb

```diff
@@ -259,14 +259,12 @@ module Bulkrax
         headers = parser.export_headers
         expect(headers).to include('id')
         expect(headers).to include('model')
-        expect(headers).to include('collections')
```

Comment:
this spec doesn't have mappings for `collection` or `file` so the numerated versions aren't in the headers either.